strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/GroupRepositoryProvider.java

```diff
@@ -88,6 +85,20 @@ public RepositoryInputStream getInputStream(String storageId,
                        .orElseGet(() -> hostedRepositoryProvider.newInputStream(
                                resolvePathTraversal(storageId, repositoryId, artifactPath)));
     }
+    
+    @Override
+    protected RepositoryInputStream newInputStream(RepositoryPath path) throws IOException
```

Comment:
Cosmetic: This `path` variable could be renamed to `repositoryPath` so that the `sPath` variable below could become `path`, making it a bit clearer.